src/Behat/MinkExtension/Context/RawMinkContext.php

```diff
@@ -158,7 +158,7 @@ public function saveScreenshot($filename = null, $filepath = null)
     {
         // Under Cygwin, uniqid with more_entropy must be set to true.
         // No effect in other environments.
-        $filename = $filename ?: sprintf('%s_%s_%s.%s', $this->getMinkParameter('browser_name'), date('c'), uniqid('', true), 'png');
+        $filename = $filename ?: sprintf('%s_%s_%s.%s', $this->getMinkParameter('browser_name'), date('Y-m-d_H.i.s'), uniqid('', true), 'png');
```

Comment:
If you can guarantee, that other components in filename (e.g. `browser_name` and `uniqueid`) won't ever contain forbidden symbols, then of course keep PR code as-as.